### PR TITLE
[CN-96] Support for scheduled backups

### PR DESCRIPTION
**Pull request by SeriyBg**

### CODE REVIEW

**Review by hasancelik — CHANGES_REQUESTED** [review]

**Review by hasancelik — APPROVED** [review]
LGTM 👍 

@SeriyBg, you marked this comment as resolved(probably accidentally :) ):
https://github.com/hazelcast/hazelcast-platform-operator/pull/124#discussion_r810983000
Could you check?

**Comment by SeriyBg** [comment]
> LGTM 👍
> 
> @SeriyBg, you marked this comment as resolved(probably accidentally :) ): [#124 (comment)](https://github.com/hazelcast/hazelcast-platform-operator/pull/124#discussion_r810983000) Could you check?

@hasancelik yeah, it was marked so after rebasing. Agree, will add this after commend.
About the cron schedule, I have put `* * * * *` as this is useful for testing since it will run every minute. Should I change it?

File: controllers/hazelcast/hot_backup_controller.go [hunk 1]
```diff
@@ -2,6 +2,15 @@ package hazelcast
 
 import (
 	"context"
+	"encoding/json"
+	"sync"
+
+	"github.com/hazelcast/hazelcast-platform-operator/controllers/util"
```
Comment (hakanmemisoglu) [inline controllers/hazelcast/hot_backup_controller.go:8]: `hazelcast-platform-operator` imports can be moved to 3rd import group

**Review by hakanmemisoglu — APPROVED** [review]
LGTM, just a small comment.

However, I would like to see two more features, maybe later on:
- Multiple schedules support instead of creating 2 HotBackup CRs
- Choice between UTC or local time zone

**Pull request merged**